src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerLinkCaseTest.java

```diff
@@ -59,29 +69,31 @@ void shouldNotConfigureLinkCaseIfFeatureFlagFalse() {
     }
 
     @Test
-    void shouldAdd2LinksToCase() {
+    void shouldSuccessfullyStayTheCase() {
```

Comment:
are we using appropriate method name here?

Reply:
Updated the name.